src/properties/peerlistwidget.cpp

```diff
@@ -153,10 +153,13 @@ void PeerListWidget::showPeerListMenu(const QPoint&)
   if (!h.is_valid()) return;
   QModelIndexList selectedIndexes = selectionModel()->selectedRows();
   QStringList selectedPeerIPs;
+  QStringList selectedPeerIPPort;
```

Comment:
no need for this. In the foreach loop insert the address:port in the selectedPeerIPs list.